plugins/azure/resoto_plugin_azure/resource/network.py

```diff
@@ -4088,6 +4088,36 @@ class AzureLoadBalancer(AzureResource, BaseLoadBalancer):
     provisioning_state: Optional[str] = field(default=None, metadata={'description': 'The current provisioning state.'})  # fmt: skip
     resource_guid: Optional[str] = field(default=None, metadata={'description': 'The resource GUID property of the load balancer resource.'})  # fmt: skip
     azure_sku: Optional[AzureSku] = field(default=None, metadata={"description": "SKU of a load balancer."})
+    aks_public_ip_address: Optional[str] = field(default=None, metadata={"description": "AKS Load Balancer public IP address."})  # fmt: skip
+
+    def post_process(self, graph_builder: GraphBuilder, source: Json) -> None:
+        def collect_p_ip_address() -> None:
+            api_spec = AzureApiSpec(
+                service="network",
+                version="2021-05-01",
+                path=self.id,
+                path_parameters=[],
+                query_parameters=["api-version", "$expand"],
```

Comment:
Can't we define this parameter when we query the list of load balancers?

Reply:
We can't, because this parameter only for individual resources. And we should provide `resource id` as a path to use additional parameters.